zenml/core/steps/split/categorical_domain_split_step.py

```diff
@@ -35,8 +35,8 @@ def lint_split_map(split_map: Dict[Text, List[CategoricalValue]]):
 def CategoricalPartitionFn(element: Any,
                            num_partitions: int,
                            categorical_column: Text,
-                           split_map: Dict[Text, List[CategoricalValue]]) \
-        -> int:
+                           split_map: Dict[Text, List[CategoricalValue]],
+                           unknown_category_policy: Text) -> int:
```

Comment:
What values can this Text take? Why does it not have a default

Reply:
As of now, PartitionFns do not need defaults because they get every argument except element and num_partitions supplied as keyword arguments from the Split Step's get_partition_fn method.